config/sync/migrate_plus.migration.su_stanford_person.yml

```diff
@@ -219,9 +240,27 @@ process:
           regex: true
         title: label/text
   su_person_mail_code: mailcode
+  su_person_academic_appt: academic_appts
+  su_person_admin_appts:
+    -
+      source: admin_appts
+      plugin: sub_process
+      process:
+        value: label/text
+  su_person_research_interests:
+    -
+      source: research_interest_topics
+      plugin: sub_process
+      process:
+        value: label/text
+  su_person_scholarly_interests/format: constants/minimal_html
+  su_person_scholarly_interests/value:
+    -
+      plugin: null_coalesce
+      source:
+        - scholarly_interests_fulltext
+        - scholarly_interests_tersetext
 destination:
-  plugin: 'entity_reference_revisions:node'
-  new_revisions: true
-  force_revision: true
```

Comment:
Do these default to false?

Reply:
Those are options only with the `entity_reference_revisions` plugin. the regular `entity` doesn't accept additional options, But basically, yes by default it does not create new revisions by using `entity` plugin.